MILESTONE.md

```diff
@@ -0,0 +1,50 @@
+# MILESTONE.md
+
+## Testnet proof-of-concept (Month 1)
+
+Description:
+Implement prototype integrated with XRPL Testnet (xls20-sandbox.rippletest.net:51233) with feature creating nft coupons and integrated with xumm app
+
+  1) https://github.com/eatozee/nftoupon
+  2) Integrate nftoupon widget to nextjs application (i.e. React ecosystem)
+
+## Public beta test (Month 2)
+
+Description:
+Refactor widget, Create collectibles component, support IOU token
+
+1) Nftoupon is the open source widget so any project owner can use it.
+2) Refactor nftoupon widget 
+3) Create collectible component and IOU token support with widget
+
+## Public beta test (Month 3)
+
+Description:
+Create a dashboard to the user to provide a api key
+
+1) Developer console dashboard
+2) Test with a batch of users
+
+## Private beta test (Month 4)
+Description:
+Integrate nftoupon widget with www.eatozee.com, https://admin.eatozee.com/#/login and eatozee mobile products
+
+1) Nftoupon widget to eatozee products
+2) Get feedback from user and clients (right now eatozee has 5 merchant)
+
+
+## Internal Organization Test (Month 5)
+
+Description: 
+Nftoupon widget compatability and smart contract checker implementation
+
+1) Make nftoupon compatible with multiple platform (right now supporting to react ecosystem)
+2) Smart contract to verify the coupon apply
+
```

Comment:
Can you please add the following bullet points for month 5
- AI to identify the Object (i.e. user will not be able to create the NFTs if it is already created or it is a bad/abusive image)
- real time streaming